public/locales/en/home.json

```diff
@@ -42,6 +42,10 @@
         "noticeCount": "{{count}} notice available"
     },
     "map": {
+        "base": {
+            "map": "Map",
```

Comment:
Google and Yandex call these layers "Map/Satellite" while Bing calls them "Road/Aerial".

I think "Map/Satelite" is the better choice.